src/app/views/query-runner/request/headers/HeadersList.tsx

```diff
@@ -42,23 +42,23 @@ const HeadersList = ({
   };
 
   const columns = [
-    { key: 'key', name: messages.Key, fieldName: 'name', minWidth: 300, maxWidth: 400 },
-    { key: 'value', name: messages.Value, fieldName: 'value', minWidth: 300, maxWidth: 400 },
-    { key: 'button', name: '', fieldName: 'button', minWidth: 200, maxWidth: 300 }
+    { key: 'key', name: translateMessage('Key'), fieldName: 'name', minWidth: 300, maxWidth: 400, ariaLabel: translateMessage('Key') },
+    { key: 'value', name: translateMessage('Value'), fieldName: 'value', minWidth: 300, maxWidth: 400, ariaLabel: translateMessage('Value') },
+    { key: 'button', name: translateMessage('actions'), fieldName: 'button', minWidth: 200, maxWidth: 300, ariaLabel: translateMessage('actions') }
```

Comment:
The `actions` key should be added to `GE.json`

Reply:
Yeah, it was already added actually